ansible_events_ui/api/__init__.py

```diff
@@ -25,6 +24,15 @@
     run_job,
     write_job_events,
 )
+from ansible_events_ui.schema.activation import (
```

Comment:
Better to keep module import here, to avoid potential name conflict with other entity types related to the same resources (e.g. declarative models, one the transition happens).

```
from ansible_events_ui import schema
```

In the `schema/__init__.py` file you can re-export the schemas from nested submodules:

```
from .activation import Activation, ActivationInstance, ActivationLog
from .extra_vars import Extravars, ExtravarsRef
# etc.
```